src/v3/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -78,7 +78,11 @@ const AuthHeader: FunctionComponent<AuthHeaderProps> = ({
         },
       }}
     >
-      <Typography variant="h1">
+      <Typography
+        variant="h1"
+        role="presentation"
+        tabIndex={-1}
```

Comment:
Is hiding the element from the tab order in addition to `role="presentation"` also necessary to prevent screenreaders from reading the element?

Reply:
As we switched from Odyssey Typography to mui Typography here, adding tabIndex=-1 to avoid potential regression, it's added in Odyssey, without this line, tabIndex attr will be removed from existing snapshots.

Code detail in Odyssey: https://github.com/okta/odyssey/blob/80b6c32e13541e5b47eec70fcb4f676cade6fda3/packages/odyssey-react-mui/src/Typography.tsx#L152